README.md

```diff
@@ -157,3 +157,50 @@ Common usage:
   ```shell
   docker run --rm -v ~/solid-config:/config -p 3000:3000 -it css:latest -c /config/my-config.json
   ```
+
+## Using the identity provider
+
+1. Launch the Community Solid Server, for example in file server mode:
+    ```bash
+    git clone git@github.com:solid/community-server.git
+    cd community-server
+    mkdir data
```

Comment:
This is actually not necessary anymore when using a file store! The directory will be created if necessary by the server.

Reply:
Removed